packages/web-service/src/pages/habitat/active-entrances/habitat-active-entrances.njk

```diff
@@ -1,37 +1,30 @@
 {% extends "standard-form.njk" %}
 
-{% from "error-summary.njk" import errorSummary %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
 {% set errorMap = {
         'habitat-active-entrances': {
-          'no-radio-selected': { ref: '#habitat-reopen', text: 'You have not selected a choice.' }
```

Comment:
I believe that the GDS standard for single sentences with no follow on text is to omit the full-stop (.).

Reply:
Thanks for explaining, resolved